src/components/common/ColonyActionsTable/hooks/useRenderRowLink.tsx

```diff
@@ -0,0 +1,31 @@
+import clsx from 'clsx';
+import React from 'react';
+
+import { type ActivityFeedColonyAction } from '~hooks/useActivityFeed/types.ts';
+import { TX_SEARCH_PARAM } from '~routes/index.ts';
+import { setQueryParamOnUrl } from '~utils/urls.ts';
+import { MEATBALL_MENU_COLUMN_ID } from '~v5/common/Table/consts.ts';
+import { type RenderCellWrapper } from '~v5/common/Table/types.ts';
+import Link from '~v5/shared/Link/index.ts';
+
+export const useRenderRowLink = (
```

Comment:
It's passed to`renderCellWrapper` which type is a function. This field is used in other tables as well and I'm not sure if it's ok to update it in this PR.

Reply:
I couldn't see `useRenderRowLink` being used anywhere else and was hoping it could be turned into a component instead of a hook, especially considering @chmanie's proposal in: https://discord.com/channels/562263648173555742/998629903740182580/1207325289227755580
> It's passed to renderCellWrapper which type is a function.

It should still be ok to pass a component to it as it is a function, right?